quesma/model/bucket_aggregations/date_histogram.go

```diff
@@ -91,6 +95,11 @@ func (query *DateHistogram) TranslateSqlResponseToJson(rows []model.QueryResultR
 		_, timezoneOffsetInSeconds := intervalStartNotUTC.Zone()
 		key -= int64(timezoneOffsetInSeconds * 1000) // seconds -> milliseconds
 
+		docCount := row.LastColValue()
+		if util.ExtractInt64(docCount) < int64(query.minDocCount) {
+			continue
+		}
```

Comment:
maybe move to the top of `for`? Also use this `docCount` below in the response map